packages/components/src/sites-table/use-sites-table-filtering.tsx

```diff
@@ -47,7 +47,7 @@ export function useSitesTableFiltering(
 	}, [ allSites, __ ] );
 
 	const filteredSites = useFuzzySearch( {
-		data: filteredByStatus[ status ],
+		data: filteredByStatus[ status ] || [],
```

Comment:
@wojtekn What do you think if we return all sites instead of an empty array? It would be like ignoring wrong statuses.

```suggestion
		data: filteredByStatus[ status ] || filteredByStatus.all,
```

Reply:
@sejas, I would treat such a case as a search phrase that is not found, so I think it makes sense to display a no results page.

If we decide to display all sites, I think that the better way would be to add a validation that happens before we filter/sort data. In such a case, we would load the "All" page explicitly - by removing the `status` param from the URL instead of just displaying all results.

What do you think?

Reply:
I love the smarter option that removes the wrong status param from the URL.
We can merge it and discuss the other implementation here pdKhl6-Au-p2